### PR TITLE
update products schema

### PR DESCRIPTION
**Pull request by rjh336**
# Description of change

Per Yotpo support RE `/v1/apps/{app_key}/products` endpoint:

> our R&D team recently changed the endpoint and the category parameter are no longer supported/deprecated the reason why it shows as null. However, we cannot delete it as it may cause some backward compatibility issues, especially with the customer who still using it. If you are not using it in some cases, kindly disregard this part of the endpoint. 

I made updates to `category.name` and `category.id` to accept null values

# Manual QA steps
 - 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Comment by rjh336** [comment]
@KrisPersonal

**Comment by dbshah1212** [comment]
> @KrisPersonal

Can you please [sign a contribution license agreement](https://stitch-cla-enforcer.herokuapp.com/) for us to look further?

**Comment by rjh336** [comment]
> > @KrisPersonal
> 
> @rjh336 Can you please [sign a contribution license agreement](https://stitch-cla-enforcer.herokuapp.com/) for us to look further?

@dbshah1212 when I click on the link and authorize I do not get redirected to a web form, nor do I receive and email to sign the CLA

**Comment by rjh336** [comment]
@KrisPersonal I can sign the form hosted in the [CLA heroku app](https://stitch-cla-enforcer.herokuapp.com/) but I do not get an email (not in spam too).

**Comment by toandm** [comment]
Hi guys, any new feedback here? I'm having the same issue, and this would help us get back to the right track here

**Comment by toandm** [comment]
@dbshah1212 please take a look at this, we really need it now. Our reports are not updating because of this

**Comment by dbshah1212** [comment]
Yes we can merge this as changes looks good to me, @rjh336 can you please reissue the CLA?

**Comment by dbshah1212** [comment]
> @dbshah1212 please take a look at this, we really need it now. Our reports are not updating because of this

Yes I can, just waiting for CLA approval

**Comment by KrisPersonal** [comment]
@rjh336  Please resubmit and CLA should get approved. https://stitch-cla-enforcer.herokuapp.com/

**Comment by rjh336** [comment]
> @rjh336 Please resubmit and CLA should get approved. https://stitch-cla-enforcer.herokuapp.com/

done @KrisPersonal

**Review by KrisPersonal — APPROVED** [review]

**Review by dbshah1212 — APPROVED** [review]

**Comment by KrisPersonal** [comment]
Completed the merge of PR to master

**Comment by toandm** [comment]
@rjh336 I don't know why but the same error is happening :( I'm using Yotpo connection on Stitch

**Comment by rjh336** [comment]
Same @toandm 

@KrisPersonal @dbshah1212 do either of you work for Stitch? Wondering if Stitch pinned their yotpo deployment to the last release of this tap and need to bump? Or my changes just did not resolve the issue?

**Comment by toandm** [comment]
nevermind it worked, thanks

**Pull request merged**
